runtime/reconcilers/metrics_view.go

```diff
@@ -132,6 +154,10 @@ func (r *MetricsViewReconciler) Reconcile(ctx context.Context, n *runtimev1.Reso
 		return runtime.ReconcileResult{Err: validateErr}
 	}
 
+	// set selectors to nil now that they are resolved
+	mv.Spec.DimensionsSelector = nil
+	mv.Spec.MeasuresSelector = nil
```

Comment:
Shouldn't this be part of the rewriting that happens in `ReconcileWithParentMetricsView`?

Reply:
yes can be done there now